### PR TITLE
Move projects to their own directory

### PR DESCRIPTION
**Pull request by illicitonion**
Commits are separately reviewable

### CODE REVIEW

File: projects/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+   <br>Learn about long-running processes, HTTP and `curl`
+1. [Servers & Databases](./server-database)
+   <br>Build a server that takes data from a database and serves it in `json` format
+1. [Multiple Servers](./multiple-servers)
+   <br>Build and run file & API servers behind nginx in a simple multi-server architecture
+1. [Docker & Cloud Deployment](./docker-cloud/)
+   <br>Use containers to reproducibly deploy applications into the cloud
+1. [Batch Processing](./batch-processing/)
+   <br>Build an image processing pipeline with cloud storage
+1. [Buggy App](./buggy-app/)
```
Comment (SallyMcGrath) [inline projects/README.md:31]: missing gRPC

File: website/build.sh [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+      ;;
+    Linux)
+      curl -L https://github.com/gohugoio/hugo/releases/download/v0.109.0/hugo_extended_0.109.0_Linux-64bit.tar.gz | tar xzf - -C "${td}" hugo
+      ;;
+  esac
+  chmod 0755 "${td}/hugo"
+  hugo="${td}/hugo"
+fi
+
+cp -r primers projects workbooks website/content/
```
Comment (SallyMcGrath) [inline website/build.sh:19]: prep is now missing

File: website/layouts/partials/page-footer.html [hunk 3]
```diff
@@ -2,23 +2,27 @@
   {{ with .Params.author}}
```
Comment (SallyMcGrath) [inline website/layouts/partials/page-footer.html:2]: let's discard all these changes too until we've sorted out our structure

File: website/layouts/_default/single.html [hunk 4]
```diff
@@ -41,7 +41,8 @@ <h2 class="c-page-header__lastmod">
 
   <section class="l-page__main c-copy">{{.Content}}</section>
 
-  {{ partial "page-footer.html" .}}
```
Comment (SallyMcGrath) [inline website/layouts/_default/single.html:44]: let's discard these changes as we are stripping frontmatter

**Review by SallyMcGrath — COMMENTED** [review]
So, I think, on reflection, that this doesn't quite make sense yet.

I think we want to keep the course stuff separate from the projects as much as possible. We want to make the projects self-contained and remixable, honouring the Choose Your Own Adventure idea. Ideally someone else could  write a different series of workbooks on a totally different schedule on their own website and still query this repo for the resources.

I don't think we should tightly couple to this run https://systems.codeyourfuture.io/versions/1-0-0/ , which makes sense for us but might not make sense for an individual not part of our programme. They don't have access to our troubleshooting yet and they don't have access to the product team unless they volunteer. I think this creates an unnecessary friction. We really want multiple ways of using the content from the start.

I think we should direct people to the website for a recommended path through, and keep the projects up front here.  Here's my suggestion:

1. move the workbooks into the website content folder and just leave the primers and projects  in the main
2. rewrite the repo readme to introduce the projects, reference the primers, and link to the website for workbooks/course info. Contributing info and prep info should remain.
3. do not copy this repo readme into website, but write a different home page for the website.

Once we're happy with the structure, snagging for me:
- change all links in workbooks
- set up edit links to github from website
- run through and check all images are working in primers
- empty website/content/primers, commit and .gitignore to keep empty

snagging for you:
- only copy markdown and images to hugo content in build script
- but keep images in their folders (eg /assets) or the links will break
- remove workbooks cp

**Comment by illicitonion** [comment]
@SallyMcGrath updated - PTAL!

File: README.md [hunk 5]
```diff
@@ -8,48 +8,20 @@ title="Projects"
 
 An immersive, introductory course to backend software engineering using [go](https://go.dev).
 
+This course is split into two strands: workbooks to read, and projects to build. In the future, there will also be troubleshooting exercises to do, and potentially product-focused work.
```
Comment (SallyMcGrath) [inline README.md:11]: Yep yep

**Review by SallyMcGrath — APPROVED** [review]
LGTM

**Pull request merged**